### PR TITLE
Sprint 3: Adds responsive styling to supervisor dashboard

### PR DESCRIPTION
**Pull request by roosdebildt**
### Sprint 3: Adds responsive styling to supervisor dashboard

- [x] Divide dashboard into responsive vertical boxes dependent on the amount of teams
- [x] Reduce the amount of text in the last status update preview
- [x] Add link to all activity of that team

@alicetragedy could you review please?

### CODE REVIEW

**Comment by alicetragedy** [comment]
this is looking good! I finally had some time to look over it today @roosdebildt and I think we can merge it. :tada:

File: app/views/supervisor/dashboard/index.html.slim [hunk 1]
```diff
@@ -1,28 +1,29 @@
...
-          h5 = t.last_activity.title
-          p = render_markdown(t.last_activity.content).html_safe
-        - else
-          p Your team hasn't written any status updates yet.
+        section.dashboard
+          h4 Activity
+          - if t.activities.any?
+            p = "Their latest update is from #{ time_ago_in_words(t.last_activity.created_at) } ago."
+            h5 = t.last_activity.title
+            p = truncate((t.last_activity.content).html_safe, length: 220) { link_to " Show all", activities_path(team_id: t.id) }
```
Comment (alicetragedy) [inline app/views/supervisor/dashboard/index.html.slim]: the `link_to` here doesn't work. 
If you want to have it all on the same line, you can put it in a block:
```
p do
= truncate(...
= link_to ..
````
Another possibility is to add it to the next line, that also works :)

**Comment by roosdebildt** [comment]
updated the link_to to newline == more consistent in style anyway == we like!

**Comment by roosdebildt** [comment]
as we've changed a lot in the meanwhile, let's first merge sprint 4 for the dashboard and this one afterwards if possible!

**Comment by alicetragedy** [comment]
@roosdebildt okay, I will take a look at sprint 4 first and then at those new changes here. :+1:

**Pull request closed without merging**